src/api/axios.ts

```diff
@@ -0,0 +1,33 @@
+import {navigate} from '@helper/RootNavigation';
+import axios from 'axios';
+
+const public_url = 'server_url';
+
+const API_URL = public_url;
```

Comment:
```suggestion
const API_URL = 'server_url';
```
이렇게 수정할 수 있지 않을까요? 따로 분리하신 이유가 있으신지 궁금합니다! 혹시 인터셉터 때문일까요?

Reply:
네 이렇게 수정해도 될거 같네요